pkgs/applications/audio/fluidsynth/default.nix

```
@@ -14,6 +14,16 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-BSJu3jB7b5G2ThXBUHUNnBGl55EXe3nIzdBdgfOWDSM=";
   };
 
+  patches = [
+    # Fixes bad CMAKE_INSTALL_PREFIX + CMAKE_INSTALL_LIBDIR concatenation for Darwin install name dir
+    # Remove when PR merged & in release
+    (fetchpatch {
+      name = "0001-Fix-incorrect-way-of-turning-CMAKE_INSTALL_LIBDIR-absolute.patch";
+      url = "https://github.com/FluidSynth/fluidsynth/pull/1261.patch";
```

please use the commit patch, because the pull patch might not be stable
```suggestion
      url = "https://github.com/FluidSynth/fluidsynth/commit/03cd38dd909fc24aa39553d869afbb4024416de8.patch";
```

---

Sure, fair enough :)